src/actions/PetitionActions.js

```diff
@@ -33,6 +33,15 @@ export function fetchPetition (id) {
   };
 }
 
+export function refreshPetition (id) {
```

Comment:
This is similar to the `fetchPetition` action but does not involve a loading state.
So this happens 'behind the scenes' for the user.